Lib/test/test_typing.py

```diff
@@ -4595,6 +4595,13 @@ class A(typing.Match):
 
 class AnnotatedTests(BaseTestCase):
 
+    def test_new(self):
+        with self.assertRaisesRegex(
+            TypeError,
+            'Type Annotated cannot be instantiated',
```

Comment:
"Type Annotated" is odd wording, it would be clearer as just "Annotated cannot be instantiated".

Reply:
I have just copy pasted the existing message. I guess it is out of scope 🙂 
Should we open a new issue?